docs/docs/outputs/output-formats.md

```diff
@@ -14,65 +13,53 @@ The following output formats are supported:
 - `webhook:http://url/webhook` - send events in json format to the webhook url
 - `none` - ignore stream of events output, usually used with --capture
```

Comment:
I think that all above CLI syntax should be removed or refactored to structured format.

Reply:
I added the one liners to each section instead -- thank you for pointing this out!

Reply:
What's a `Fluent receiver.`?